loadrunner_professional/assets/logs/loadrunner_professional.yaml

```
@@ -0,0 +1,62 @@
+id: loadrunner
+metric_id: loadrunner-professional
+backend_only: false
+facets: null
+pipeline:
+  type: pipeline
+  name: LoadRunner Scenario Information Pipeline
+  enabled: true
+  filter:
+    query: "source:loadrunner"
+  processors:
+    - type: attribute-remapper
+      name: Map attribute `hostname` to tag `host`
+      enabled: true
+      sources:
+        - hostname
+      sourceType: attribute
+      target: host
+      targetType: tag
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map attribute `scenario_started.general.version`, `scenario_ended.general.version` to tag `version`
+      enabled: true
+      sources:
+        - scenario_started.general.version
+        - scenario_ended.general.version
+      sourceType: attribute
+      target: version
+      targetType: tag
+      preserveSource: true
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map attribute `scenario_started.summary.mode` to tag `loadrunner.scenario.mode`
+      enabled: true
+      sources:
+        - scenario_started.summary.mode
+      sourceType: attribute
+      target: loadrunner.scenario.mode
+      targetType: tag
```

Is there any reason to keep this as a tag ? While host and version are standard tags, I get the impression that the other attributes being remapped here look more like attributes (tags are mostly infra related)
```suggestion
      targetType: attribute
```